projects/bp-gallery/src/components/views/exhibitions/ExhibitionOverview.tsx

```diff
@@ -0,0 +1,398 @@
+          </Button>
+          {isCurator && (
+            <Button
+              onClick={e => {
+                e.stopPropagation();
+                visit(editLink);
+              }}
+            >
+              {t('common.edit')}
+            </Button>
+          )}
+        </div>
+      </div>
+      {isCurator && (
+        <div className='absolute top-2 right-2 bg-white rounded-full'>
+          <IconButton onClick={() => setIsPopupOpen(true)}>
+            <Delete />
+          </IconButton>
+        </div>
+      )}
+    </div>
+  );
+};
+
+const ExhibitionSmallCard = ({
+  isCurator,
+  link,
+  editLink,
+  titlePictureLink,
+  exhibitionTitle,
+  setIsPopupOpen,
+}: {
+  isCurator: boolean;
+  link: string;
+  editLink: string;
+  titlePictureLink: string;
+  exhibitionTitle: string;
+  setIsPopupOpen: Dispatch<SetStateAction<boolean>>;
+}) => {
+  const { t } = useTranslation();
+  const { visit } = useVisit();
+  const { isMobile } = useContext(MobileContext);
+  return (
+    <div className={`${isMobile ? 'w-full' : 'w-[20rem] p-4'}`}>
+      <Card
+        className='relative'
+        onClick={() => {
+          !isCurator && visit(link);
+        }}
+      >
+        <CardActionArea disableRipple={isCurator}>
+          <CardMedia
+            component='img'
+            height='200'
+            image={titlePictureLink}
+            alt='exhibition picture'
+          />
+          <CardContent
+            sx={{ height: isCurator ? '4rem !important' : '2rem !important' }}
+            className='flex flex-col justify-between'
+          >
+            <div className='text-xl font-bold line-clamp-1'>{exhibitionTitle}</div>
+            {isCurator && (
+              <div className='flex gap-2 flex-row-reverse'>
+                <Button
+                  onClick={e => {
+                    e.stopPropagation();
+                    visit(link);
+                  }}
+                >
+                  {t('exhibition.overview.to-exhibition')}
+                </Button>
+                <Button
+                  onClick={e => {
+                    e.stopPropagation();
+                    visit(editLink);
+                  }}
+                >
+                  {t('common.edit')}
+                </Button>
+              </div>
+            )}
+          </CardContent>
+        </CardActionArea>
+        {isCurator && (
+          <div className='absolute top-2 right-2 bg-white rounded-full'>
+            <IconButton onClick={() => setIsPopupOpen(true)}>
+              <Delete />
+            </IconButton>
+          </div>
+        )}
+      </Card>
+    </div>
+  );
+};
+
+const DeleteModal = ({
+  isPopupOpen,
+  setIsPopupOpen,
+  exhibitionTitle,
+  exhibitionId,
+  refetch,
+}: {
+  isPopupOpen: boolean;
+  setIsPopupOpen: Dispatch<SetStateAction<boolean>>;
+  exhibitionTitle: string;
+  exhibitionId: string;
+  refetch: any;
+}) => {
+  const [deleteExhibition] = useDeleteExhibitionMutation();
```

Comment:
It's probably much easier to use the refetchQueries option of useDeleteExhibitionMutation here instead of passing refetch as prop through multiple components (similar to how it's done in e.g. FormattedComment)